spec/models/manager_refresh/inventory_collection_default_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+describe ManagerRefresh::InventoryCollectionDefault do
+  it 'pickups existing Vm and MiqTemplate records on Amazon EMS re-adding' do
```

Comment:
@AlexanderZagaynov can we move this spec into amazon repo? ```ems_amazon_with_vcr_authentication``` would create cyclical dependecy

Reply:
it you would like to test it here, we would do just persitor spec, without VCRs

some thing like https://github.com/Ladas/manageiq/blob/19e7b46b2769b4f5c64601232586dfd98875b62f/spec/models/manager_refresh/persister/local_db_finders_spec.rb#L150

just create a mock persister, include there the InventoryCollection of interest, build some mock data, call perister.persist! and check it reused the Vm that was already in the db

Reply:
@Ladas AWS specs removed